routes/index.ts

```
@@ -41,5 +41,6 @@ app.use("/v1/notifications", require("./notify"));
 app.use("/goals", require("./goals"));
 app.use("/invites", require("./invites"));
 app.use("/requests", require("./requests"));
+app.use("/impersonation", require("./impersonation"))
```

ESLint

[error] Insert `;`

(prettier/prettier)